services/frontend-service/src/ui/utils/Links.tsx

```diff
@@ -152,12 +162,12 @@ export const DisplaySourceLink: React.FC<{ displayString: string; commitId: stri
     return null;
 };
 
-export const DisplayManifestLink: React.FC<{ displayString: string; app: string; version: number }> = (
+export const DisplayManifestLink: React.FC<{ displayString: string; app: string; version: ReleaseNumbers }> = (
     props
 ): JSX.Element | null => {
     const { displayString, app, version } = props;
 
-    let manifestLink = '/ui/manifest?app=' + app + '&release=' + version;
+    let manifestLink = '/ui/manifest?app=' + app + '&release=' + version.version + '&revision=' + version.revision;
```

Comment:
i think you need to check here if revisionsEnabled==true, otherwise it would break for setups without revisions.